project/Vagrantfile

```diff
@@ -65,13 +69,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         master.vm.box = "chef/centos-6.5"
         master.vm.provider "virtualbox" do |v|
             v.name = mastername
-            v.memory = 2048
-            v.cpus = 2
+            v.memory = @memory
+            v.cpus = @cpus
         end
         master.vm.hostname = mastername
         master.vm.network "public_network", bridge: 'en0: Wi-Fi (AirPort)'
+        #master.vm.network "private_network", type: "dhcp"
         master.vm.synced_folder "/opt/vagrant", "/opt/vagrant"
         master.vm.synced_folder "/space/software", "/space/software"
+        master.vm.synced_folder Dir.getwd, "/project/root"
```

Comment:
it starts from the current folder and keeps on going up in the directory
tree until it find one VagrantFile :)
Check  https://docs.vagrantup.com/v2/vagrantfile/index.html (Lookup path)

On Mon, Feb 23, 2015 at 2:50 PM, Geert <notifications@github.com> wrote:

> In project/Vagrantfile
> <https://github.com/grtjn/mlvagrant/pull/21#discussion_r25164055>:
>
> >          master.vm.synced_folder "/opt/vagrant", "/opt/vagrant"
> >          master.vm.synced_folder "/space/software", "/space/software"
> > +        master.vm.synced_folder Dir.getwd, "/project/root"
>
> Didn't know you could run vagrant up in a different dir than Vagrantfile.
> How does it know where to look for it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grtjn/mlvagrant/pull/21/files#r25164055>.
>